dill/_shims.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+#
+# Author: Mike McKerns (mmckerns @caltech and @uqfoundation)
+# Copyright (c) 2008-2016 California Institute of Technology.
```

Comment:
Feel free to add an Author line for yourself here.  Also, as this is a new module, it is acceptable to only copyright to the current year.

Reply:
That's better, thanks.